tobago-core/src/main/java/org/apache/myfaces/tobago/internal/renderkit/renderer/SheetRenderer.java

```diff
@@ -600,17 +606,7 @@ private void encodeTableBody(
         // todo like in TreeListboxRenderer
         writer.writeAttribute(DataAttributes.TREE_PARENT, parentId, false);
       }
-
-      AbstractUIRow row = null;
-      for (final UIColumn column : columns) {
-        if (column.isRendered()) {
-          if (column instanceof AbstractUIRow) {
-            row = (AbstractUIRow) column;
-            // todo: Markup.CLICKABLE ???
-          }
-        }
-      }
-      // the row client id depends from the existence of an UIRow component! TBD: is this good?
+      // the row client id depends on from the existence of an UIRow component! TBD: is this good?
```

Comment:
The client ID of a row depends on the presence of a UIRow component. TBD: is this good?